src/calendars/calendars.jl

```diff
@@ -58,6 +58,8 @@ struct NullHolidayCalendar <: HolidayCalendar end
 isholiday(::NullHolidayCalendar, dt::Dates.Date) = false
 isbday(::NullHolidayCalendar, dt::Dates.Date) = true
 bdays(::NullHolidayCalendar, dt0::Dates.Date, dt1::Dates.Date) = dt1 - dt0
```

Comment:
Please, check that `bdays(::NullHolidayCalendar, dt0::Dates.Date, dt1::Dates.Date)` can be removed without affecting performance in this case.